### DISCUSSION TITLE
Switch the content of the "Grid Row" and "Grid Column" docs

### DISCUSSION THREAD

**Discussion by momander** [quasarframework/quasar#12774] [Ideas / Proposals]
It took me a long time to understand column width in Quasar. I now realize it is because I looked at the "Grid Column" page (https://quasar.dev/layout/grid/column) to find documentation for column width and didn't find it. The documentation of column widths is on the "Grid Row" page instead. It seems to me like the contents of the "Grid Row" page is mostly about columns, and the contents on the "Grid Column" page is mostly about rows.

I propose moving some of the sections between these two pages to make them easier to understand. The new sections on each page would be as follows.

**"Grid Row" page -- https://quasar.dev/layout/grid/row**
Equal-height
Setting one row height
Variable height content
Vertical alignment
Link to Flex Grid Playground

**"Grid Column" page -- https://quasar.dev/layout/grid/column**
Equal-width
Setting one column width
Variable width content
Horizontal alignment
Responsive classes
Link to Flex Grid Playground

I am worried that the current organization of row/column content is making Quasar harder to understand than it has to be, and holding back adoption. What do you think?

**Comment by sushi2all** [comment]
Second that. I found the documentation on grid management confusing too and finally got to understand it by reading this post 😄

**Comment by pdanpdan** [comment]
Row and column are the containers
